lib/jscsrc-builder.js

```diff
@@ -0,0 +1,60 @@
+/* jshint node: true */
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var temp = require('temp');
+
+temp.track();
+
+function getCustomConfig() {
+  if (fs.existsSync('.jscsrc')) {
+    var contents = fs.readFileSync('.jscsrc', { encoding: 'utf8' });
+
+    return JSON.parse(contents);
+  } else {
+    return {};
+  }
+}
+
+function mergeConfigs(userConfig, suaveConfig) {
+  var config = {};
+  var keys, index, length, key;
+
+  keys = Object.keys(suaveConfig);
+  for (index = 0, length = keys.length; index < length; index++) {
+    key = keys[index];
+
+    config[key] = suaveConfig[key];
+  }
+
+  keys = Object.keys(userConfig);
+  for (index = 0, length = keys.length; index < length; index++) {
+    key = keys[index];
+
+    if (config[key] === null) {
```

Comment:
Did you mean `if (userConfig[key] === null)`?

So if the default config has `"requireEnhancedObjectLiterals": true` and users wish to disable that custom rule, they would set `"requiredEnhancedObjectLiterals": null` in their own `.jscsrc`. And I think the intention here was to delete that rule completely from the final config object?

As written, the final config would still contain `"requiredEnhancedObjectLiterals": null`, though interestingly enough, JSCS seems to ignore the rule (guessing because the value is `null`). So it doesn't fail the [`truthy`](https://github.com/dockyard/ember-suave/blob/v0.1.6/lib/rules/require-enhanced-object-literals.js#L7) check. In fact, you end up with the same desired effect: not checking for the rule at all.